src/cli/tls_utils.cpp

```diff
@@ -140,7 +140,8 @@ class TLS_Client_Hello_Reader final : public Command
 
          try
             {
-            Botan::TLS::Client_Hello hello(input);
+            // TODO: deal with Client_Hello_13
```

Comment:
- [ ] allow printing `Client_Hello_13` infos in cli utils

We'll need to allow deserializing a Client Hello w/o knowing if it'll be 1.2 or 1.3, just as we'll eventually need to do for the TLS 1.3 server.

Reply:
> We'll need to allow deserializing a Client Hello w/o knowing if it'll be 1.2 or 1.3, just as we'll eventually need to do for the TLS 1.3 server.

I feel, it would be fine to postpone that until the TLS 1.3 server is implemented.